content/resources/cao/index.en.md

```diff
@@ -9,19 +9,19 @@ layout: "single-resource"
 
 ## Terminology
 
-The whole sphere of collective labour agreements is often filled with specialist jargon. Here's an overview to clarify the main terms.  
-
-Dutch | English | Description 
---- | --- | ---
-*Algemeen verbindend verklaring* (AVV) | Generally binding agreement | If a collective labour agreement is generally binding, then the agreement holds for all workers in a role who meet specific conditions. Employing companies are bound by generally binding agreements and can't opt out. 
-*Bedrijfstak* or *sector* | Industry | For example 'tech industry', 'manufacturing industry', 'hospitality industry'.
-*Beroep* | Profession | The work or role you do in exchange for wages or other forms of payment.
-_Collectieve arbeidsovereenkomst_ (CAO) | Collective labour agreement | A contract that is usually signed between one or more employer, and one or more worker organisation 
-_Onderneming_ | Enterprise, venture, or company | In the context of collective labour agreeents and works councils, the term generally refers to an employing company as an entity.
-_Ondernemingsraad_ (OR) | Works council | Dutch law requires organisations of 50 or more workers to have a works council. Smaller organisations can also have a works council if they elect to do so. Through the works council, representatives from the company talk directly with the boss.
-*Vakbond* | Workers' union | A workers’ union is a set of its members, the workers who have joined the union.
-*Werkgeversorganisatie* or *werkgeversvereniging* | Employer organisation, employer's union | There are at least two forms of employer unions: <ul><li>A collective of companies, so that they can be represented at a government level, for example as is the case with the collective of small and medium-sized companies (_midden- en kleinbedrijf_, MKB)</li><li>A regular union</li> </ul>
-*Wet open overheid* (WOO) | Law open government | A law that entitles people to get information from the government on any government operations and decisions. In some countries this is also known as a freedom of information act.
+The whole sphere of collective labour agreements is often filled with specialist jargon. Here's an overview to clarify the main terms.
```

Comment:
Prettier likes to make tables differently. The content looks the same on the frontend though.